examples/emotion/rspack.config.js

```diff
@@ -1,3 +1,9 @@
+const { createSwcLoaderExperiments } = require("@rspack/cli");
+
+const rspackExperiments = createSwcLoaderExperiments()
```

Comment:
Is `createSwcLoaderExperiments` only exists for migrate? Could we avoid add this to core? Maybe add to a separate package?

Reply:
It's for convenience and might be existing for a long run. Creating a new package maybe too much for this case?

Reply:
IMO, core shouldn't contain this, and if we want to remove it from core in the future we will have to make a break change

and if swc's raw options for these visitor is not convenience enough, we can introduce a new stage to normalize/default the options before send raw options to rust side, maybe now is a good time since we already have to make some break

Reply:
Remove the normalization part completely from user side.